docs/README.md

```diff
@@ -50,6 +50,8 @@ Content that is indexed in the search index by default:
 - Terms
 - Term Meta
 
+**Note:** Post meta that is "protected"--i.e. has a key beginning with `_`--will not be indexed automatically. To index these fields, use the `ep_prepare_meta_allowed_protected_keys` filter. It will accept a value of boolean `true` (which will index *all* protected meta) or an array containing the keys of specific protected meta fields you want to index.
```

Comment:
```suggestion
**Note:** Post meta that is "protected" - i.e. has a key beginning with `_` - will not be indexed automatically. To index these fields, use the `ep_prepare_meta_allowed_protected_keys` filter. It will accept a value of boolean `true` (which will index *all* protected meta) or an array containing the keys of specific protected meta fields you want to index.
```

Reply:
It is worth noting that protected meta keys doesn't necessarily start with an underscore ?

Reply:
ie, replacing `ie` with `eg`